docs/md/workload.md

```diff
@@ -0,0 +1,67 @@
+# Workloads
+
+## What is it?
+Within the context of Workflow Launcher, a workload is a discrete body of work, which takes data from a source, pushes it into a workflow executor for analysis, and then delivers the results of the analysis to an output location (also known as a sink).
+
+## Workload Components
+### Source
+The workload [Source](./source.md) models the first stage of a processing pipeline. In a
+typical workload configuration, a `Source` can be used to read workflow inputs
+from a specified location or service in the cloud.
+
+### Executor
+The workload [Executor](./executor.md) models an intermediate stage of a processing pipeline.
+In a typical workload configuration, an `Executor` uses a supported
+service in the cloud to execute workflows.
+
+### Sink
+The workload [Sink](./sink.md) models the terminal stage of a processing pipeline. In a
```

Comment:
The other links work (woo!) and this one will 404 until this PR is merged: https://github.com/broadinstitute/wfl/pull/449

Reply:
Yep - that was what I expected.